src/wwwroot/app/services/ThesaurusService.js

```diff
@@ -52,6 +54,19 @@ export default class ThesaurusService {
       }
    }
 
+   getThesaurusTopicsByIds(thesaurusName, arrIds) {
+      let isArrayIds = isArray(arrIds);
+      return this.getThesaurusTopics(thesaurusName).then((topics) => {
+         return filter(topics, (topic) => {
```

Comment:
This function isn't helper function for this service. I use it in VacancyService for getting user by his ID